src/views/List.jsx

```diff
@@ -16,21 +17,28 @@ export function List({ data }) {
 
 	return (
 		<>
-			<form>
-				<label htmlFor="filter-items">Filter items</label>
-				<input
-					type="search"
-					name="filter-items"
-					id="filter-items"
-					onChange={onSearchChange}
-					value={searchField}
-				/>
-				<ul>
-					{filteredLists.map(({ name, id }) => (
-						<ListItem key={id} name={name} />
-					))}
-				</ul>
-			</form>
+			{filteredLists[0] ? (
```

Comment:
Here you check whether any items exist in `filteredLists` . But this has a drawback: The `filteredLists` array is also empty when you search for an item that doesn't exist on the list. If you do a search, and the item doesn't exist in the list, you will get the prompt telling you that your list is empty, which is not true.
So here it's better to check `data` to see if there are any items in the list.
Also `filteredLists` is not such a good variable name because it implies many lists exist. A better name would be `filteredList` or `filteredListItems`.